common/lib/xmodule/xmodule/modulestore/split_mongo/split_mongo_kvs.py

```diff
@@ -148,7 +148,10 @@ def has(self, key):
             if key.scope == Scope.children:
                 return False
 
-            b_type = key.block_scope_id.block_type
+            try:
+                b_type = key.block_scope_id.block_type
```

Comment:
Ah. That looks like a bug in the `split_mongo_kvs`. In particular, (as per your screenshot), the scope being used for this field is `Scope(user=UserScope.NONE, block=BlockScope.TYPE)`. In that scope, the `block_scope_id` should just be the block type (which it is, in this example). I think your best bet in this case would be to raise an error if the key scope isn't one of `Scope.content` or `Scope.settings` (as I think those are the only scopes that split-mongo handles right now).